cli/create/builtin_templates/templates/config_storage/config.yaml.tt.template

```diff
@@ -0,0 +1,50 @@
+credentials:
+  users:
+    replicator:
+      password: 'topsecret'
+      roles: [replication]
+    client:
+      password: 'secret'
```

Comment:
Could we parametrize the username and the password?

Reply:
And what about `replicator` username/password?

Reply:
Added parametrizarion of username/password.